src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/RichTextEditorIT.kt

```diff
@@ -0,0 +1,144 @@
+package com.atlassian.performance.tools.jiraactions.api.scenario
+
+import com.atlassian.performance.tools.dockerinfrastructure.api.browser.DockerisedChrome
+import com.atlassian.performance.tools.dockerinfrastructure.api.jira.JiraCoreFormula
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import com.atlassian.performance.tools.jiraactions.api.ActionResult
+import com.atlassian.performance.tools.jiraactions.api.SeededRandom
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.action.*
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.measure.output.CollectionActionMetricOutput
+import com.atlassian.performance.tools.jiraactions.api.memories.User
+import com.atlassian.performance.tools.jiraactions.api.memories.UserMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveIssueKeyMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveIssueMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveJqlMemory
+import com.atlassian.performance.tools.jiraactions.api.memories.adaptive.AdaptiveProjectMemory
+import com.atlassian.performance.tools.jiraactions.api.w3c.DisabledW3cPerformanceTimeline
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import java.time.Clock
+import java.util.*
+
+/**
+ * Test Editing issue without SetupAction (i.e Rich Text editor enabled)
+ * Covers:
+ *      EditIssue
+ *      AddComment
+ */
+class RichTextEditorIT {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    @Test
+    fun shouldRunScenarioWithoutErrors() {
+        val port = 8080
+        val version = "8.0.0"
```

Comment:
So we're sticking with 8.0.0 only. That's fine, but I want to make sure it's deliberate.

Reply:
Yes. I just want to merge it and reduce our work in progress. I don't have the capacity to make the test parallel and I don't want to make our dev loop slower (by running the test in a sequence).

Reply:
*If* we're skipping Jira 7.2.0/7.3.0 (see `RichTextEditorConfiguration` comments), then it's a bit risky. Note that this change is not opt-in and there's no easy opt-out (apart from never bumping). This PR changes how the `EditIssuePage` and `CommentForm` work and if the RTE detection doesn't work on 7.2.0/7.3.0, we're breaking them.

Reply:
You're right. I'm not sure if it's worth pulling the change.

Reply:
If can we cheaply run it for 7.2.0, 7.3.0 and 8.0.0 in parallel, then it's worth it IMHO.

Reply:
I've got another idea, but it could be controversial. We could choose the version randomly. None of them will be favoured. We can also schedule daily build to see if it flakes on some versions.

The problem of compatibility with a wide range of versions is not a problem of this test. The whole module should be tested with different Jira versions.

Reply:
I'd rather test 8.0.0 only than test random versions.

Reply:
Agreed with @sebapawlak 
Randomness can lead to us merging a bug that manifests itself on 7.3.0 and if the PR CI skips over it randomly, we'll get a fail late, perhaps even after a release.
IMHO parallielization complexity is worth it and we've been doing it a lot already, it works.

> The whole module should be tested with different Jira versions.

Maybe. But we can do it without randomness. E.g. we can add Gradle tasks per Jira version, which would set a system property, read by the tests (with a default).
However, we don't need to do it in this PR. It's big enough already.